tests/backup/backup_test.go

```diff
@@ -2661,6 +2663,244 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
+
+				userBackups, err := GetAllBackupsForUser(username, password)
+				log.FailOnError(err, "Fail on Fetching backups for %s", username)
+
+				restoreName := fmt.Sprintf("%s-%v", RestoreNamePrefix, time.Now().Unix())
+				ValidateSharedBackupWithUsers(username, ctxNonAdmin, accessLevel, userBackups[len(userBackups)-1], restoreName)
+				if accessLevel != ViewOnlyAccess {
+					restoreNames = append(restoreNames, restoreName)
+				}
+				log.Infof("RestoreNames - %v", restoreNames)
+				if accessLevel == FullAccess {
+					log.Infof("The full access exit begins")
+					break
+				}
+				//wait for 15 minutes
+				log.InfoD("wait for 15 minutes")
+				time.Sleep(15 * time.Minute)
+				fetchedUserBackups, err := GetAllBackupsForUser(username, password)
+				log.FailOnError(err, "Fail on Fetching backups for %s", username)
+				log.Infof("All the backups for user %s - %v", username, fetchedUserBackups)
+
+				recentBackupName := fetchedUserBackups[len(fetchedUserBackups)-1]
+				log.Infof("recent backup - %v ", recentBackupName)
+
+				//Check if Schedule Backup came up or not
+				dash.VerifyFatal(len(fetchedUserBackups), len(userBackups)+1, "Verifying the new schedule backup is up or not")
+
+				//Now get the status of new backup -
+				backupSuccessCheck := func() (interface{}, bool, error) {
+					backupDriver := Inst().Backup
+					bkpUid, err := backupDriver.GetBackupUID(ctx, recentBackupName, orgID)
+					if err != nil {
+						return "", true, err
+					}
+					backupInspectRequest := &api.BackupInspectRequest{
+						Name:  recentBackupName,
+						Uid:   bkpUid,
+						OrgId: orgID,
+					}
+					resp, err := backupDriver.InspectBackup(ctx, backupInspectRequest)
+					if err != nil {
+						return "", true, fmt.Errorf("unable to fetch inspect backup response for %v", recentBackupName)
+					}
+					actual := resp.GetBackup().GetStatus().Status
+					expected := api.BackupInfo_StatusInfo_Success
+					if actual != expected {
+						return "", true, fmt.Errorf("backup status for [%s] expected was [%s] but got [%s]", recentBackupName, expected, actual)
+					}
+					return "", false, nil
+				}
+				_, err = task.DoRetryWithTimeout(backupSuccessCheck, 10*time.Minute, 30*time.Second)
+				log.FailOnError(err, "Backup with name %s was not successful", recentBackupName)
+				log.InfoD("New backup - %s is successful from schedule backup ", recentBackupName)
+			}
+			log.InfoD("All Accesses are toggled and operations are performed")
+
+		})
+
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		log.InfoD("End test")
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+
+		//Get scheduleBackupUid
+		log.Infof("Get scheduleBackupUid")
+		scheduleBackupUid, err := GetScheduleUID(backupName, orgID, ctx)
+		log.InfoD("scheduleBackupUid - %v", scheduleBackupUid)
+		log.FailOnError(err, "Error in fetching schedule UID for %v", backupName)
```

Comment:
Let's print the `log.InfoD()` after the `log.FailOnError()`, because if it fails `GetScheduleUID()` there going to be no `scheduleBackupUid`